nntrainer/tensor/manager.cpp

```diff
@@ -567,17 +529,7 @@ void Manager::allocateDerivatives() {
 void Manager::deallocateDerivatives() {
   shared_deriv.deallocate();
 
-  if (LAYER_V2) {
-    for (auto &in : inputs_v2) {
-      in->deallocateGradient();
-    }
-    for (auto &out : outputs_v2) {
-      out->deallocateGradient();
-    }
-    // for (auto &t : tensors_v2) {
-    //   t->deallocateGradient();
-    // }
-  } else {
+  if (!LAYER_V2) {
```

Comment:
How about deleting this as well (maybe in later PR??)

Reply:
its cleaned up right in next PR :)

Reply:
Awesome :)